Mathlib/RingTheory/Flat/Basic.lean

```diff
@@ -106,19 +114,20 @@ lemma iff_rTensor_injective :
 inclusion `I → R` and the identity `M → M` is injective. See `iff_rTensor_injective` to
 restrict to finitely generated ideals `I`. --/
 theorem iff_rTensor_injective' :
-    Flat R M ↔ ∀ I : Ideal R, Injective (rTensor M I.subtype) := by
+    Flat R M ↔ ∀ I : Ideal R, Function.Injective (rTensor M I.subtype) := by
```

Comment:
Why have you removed `open Function`?

Reply:
Yes, because Injective objects and `Function.Injective` are in conflict here.

Reply:
I see, no problem.